LazyProxy/LazyProxyBuilder.cs

```diff
@@ -220,7 +220,8 @@ private static TypeBuilder AddMethods(this TypeBuilder typeBuilder, Type type, F
                     generator.Emit(OpCodes.Ldarg, i);
                 }
 
-                generator.Emit(OpCodes.Call, method);
+                generator.Emit(OpCodes.Tailcall);
+                generator.Emit(OpCodes.Callvirt, method);
```

Comment:
Do you have any information why it has ceased to be possible at .NET Core 3?

Reply:
I tried to search info but unsuccessfully.
Guessing that all interface implementation functions are virtual now.